components/brave_rewards/browser/rewards_service.cc

```diff
@@ -61,6 +61,9 @@ void RewardsService::RegisterProfilePrefs(PrefRegistrySimple* registry) {
   registry->RegisterUint64Pref(prefs::kStateServerPublisherListStamp, 0ull);
   registry->RegisterStringPref(prefs::kStateUpholdAnonAddress, "");
   registry->RegisterStringPref(prefs::kRewardsBadgeText, "1");
+#if defined(OS_ANDROID)
```

Comment:
not sure if we need to wrap this as `kUseRewardsStagingServer` is used all across for android without wrapper. Just a nit

Reply:
I think it's only applied for android and seems like it should be deprecated anyway